lib/still/web/router.ex

```diff
@@ -17,15 +17,22 @@ defmodule Still.Web.Router do
   end
 
   get "*path" do
-    file = "#{Path.join(get_output_path(), Path.join(path))}.html"
+    if not send_file(conn, "#{Path.join(get_output_path(), Path.join(path))}/index.html") and
+         not send_file(conn, "#{Path.join(get_output_path(), Path.join(path))}.html") do
+      conn
+      |> send_resp(404, "File not found")
+    end
+  end
 
+  defp send_file(conn, file) do
     if File.exists?(file) do
       conn
       |> put_resp_header("Content-Type", "text/html; charset=UTF-8")
       |> send_file(200, file)
+
+      true
```

Comment:
Is this working? You ignore lose the `conn` object.

Reply:
Yap, it works. After `send_file` nothing happens anyway.

Reply:
Updated